src/bin/deku_cli.ml

```diff
@@ -358,21 +366,13 @@ let originate_contract =
   Term.(
     lwt_ret
       (const originate_contract
-      $ folder_node
+      $ folder_node 0
```

Comment:
I don't understand why `0` 😕 ...

Reply:
Because it's a positional argument `Cmdliner.Arg.pos`, which requires us to specify the expected position of the argument. in this case it's the 0th argument.